test/cli.js

```diff
@@ -290,3 +292,89 @@ test('handles NODE_PATH', function (t) {
 		t.end();
 	});
 });
+
+test('Show `expected` and `actual` without a message', function (t) {
+	execCli('fixture/assert-equal-without-message.js', function (err) {
+		t.ok(err);
+
+		var runStatus = {
+			formatAssertionError: RunStatus.prototype.formatAssertionError
+		};
+
+		var assertionError = {
+			actual: 4,
+			expected: 3,
+			name: 'AssertionError',
+			message: '3 == 4'
+		};
+
+		var output = runStatus.formatAssertionError(assertionError);
+
+		compareLineOutput(t, output, [
+			'',
+			'    ' + chalk.red('AssertionError: 3 == 4'),
+			chalk.green('    + expected') + chalk.red(' - actual'),
+			'',
+			chalk.red('    -4'),
+			chalk.green('    +3')
+		]);
+
+		t.end();
+	});
+});
+
+test('Show `expected` and `actual` with a message', function (t) {
+	execCli('fixture/assert-equal-with-message.js', function (err) {
+		t.ok(err);
+
+		var runStatus = {
+			formatAssertionError: RunStatus.prototype.formatAssertionError
+		};
+
+		var assertionError = {
+			actual: 4,
+			expected: 3,
+			name: 'AssertionError',
+			message: 'foo'
+		};
+
+		var output = runStatus.formatAssertionError(assertionError);
```

Comment:
We are ignoring the output of the cli run here?

We should be comparing this to `stdout`, not generating a new output.